util/close.go

```diff
@@ -14,7 +14,7 @@ import (
 // ensure that the connection is freed. If the body is not read and closed, a
```

Comment:
Since you touch this file, could you remove the superfluous `p` at the end of line 3? 😄

Reply:
Haha sure, this PR is already a monster.. whats one more character? 😄